consensus/expected.go

```diff
@@ -54,10 +54,6 @@ const ECV uint64 = 10
 // ECPrM is the power ratio magnitude defined in the EC spec.
 const ECPrM uint64 = 100
 
-// LookBackParameter is the protocol parameter defining how many blocks in the
-// past to look back to sample randomness values.
-const LookBackParameter = 3
```

Comment:
I had to move this in order to prevent an import cycle when writing the chain-sampling function. It ended up in an unfortunate place, but I wanted to keep this diff small.